### ISSUE TITLE
Make EXEC_DIR optional

### ISSUE THREAD

**Issue by KerstenBreuer** [CompEpigen/CWLab#50]
Currently, for each job an exec dir is created that contains:
- logs
- output folders
- input parameters as yaml
- an optional directory for uploaded input data

Make the use of this directory optional and transfer everything to the database/project manager.

**Comment by KerstenBreuer** [comment]
Low priority as this doesn't prevent us from moving forward. However, should be solved for high-scale production use.

**Comment by KerstenBreuer** [comment]
depends on #49 #43 #45
